marlowe-playground-client/src/Simulation.purs

```diff
@@ -686,6 +681,37 @@ inputItem isEnabled person NotifyInput =
         [ text "+" ]
     ]
 
+inputItem state isEnabled person (MoveToSlot slot) =
+  div
+    [ classes [ aHorizontal, ClassName "flex-wrap" ] ]
+    ( [ div []
+          [ p [ class_ (ClassName "choice-input") ]
+              [ spanText "Move to slot "
+              , marloweActionInput isEnabled (SetSlot <<< wrap) slot
+              ]
+          , p [ class_ (ClassName "choice-error") ] error
+          ]
+      ]
+        <> addButton
+    )
+  where
+  addButton =
+    if isEnabled && inFuture then
+      [ button
+          [ classes [ plusBtn, smallBtn, ClassName "align-top" ]
+          , onClick $ const $ Just $ MoveSlot slot
+          ]
+          [ text "+" ]
+      ]
+    else
+      []
+
+  inFuture = state ^. (_currentMarloweState <<< _slot <<< to (greaterThan slot))
```

Comment:
Are you sure this is better than:

`inFuture = view (_currentMarloweState <<< _slot) state > slot`

Reply:
I'm not sure of anything any more